core/src/solve.jl

```diff
@@ -764,9 +749,9 @@ function continuous_control!(
 
         # When the controlled pump flows out into fractional flow nodes
         if controls_pump
-            for id in outneighbors(graph_flow, controlled_node_id)
+            for id in outneighbor_labels_type(graph, controlled_node_id, EdgeType.flow)
```

Comment:
Slightly less generic, but how about creating a `outflow_ids` function for this, since we use it so much? Together with `inflow_ids`. We can even make `<in/out>flow_id` that do `only(x)` for us.